packages/components/group/index.tsx

```diff
@@ -12,12 +12,25 @@ import { LandingPageLayout } from '@acter/components/group/layout'
 import { DescriptionSection } from '@acter/components/group/sections/description'
 import { LinksSection } from '@acter/components/group/sections/links'
 import { MembersSection } from '@acter/components/group/sections/members'
+import { SettingsSection } from '@acter/components/group/sections/settings'
 import { UpcomingActivities } from '@acter/components/group/sections/upcoming-activities'
 import { PostList } from '@acter/components/posts'
+import { useActer } from '@acter/lib/acter/use-acter'
+import { useUser } from '@acter/lib/user/use-user'
+import { userHasRoleOnActer } from '@acter/lib/user/user-has-role-on-acter'
+import { ActerConnectionRole } from '@acter/schema'
 
 export const GroupLanding: FC = () => {
   const classes = useStyles()
 
+  const { user } = useUser()
+  const { acter } = useActer()
+
+  if (!acter) return null
+
+  const isAdmin = userHasRoleOnActer(user, ActerConnectionRole.ADMIN, acter)
+  if (!isAdmin) return null
```

Comment:
are we not showing group landing page if the user is not `admin` or for the guests ?